.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "python.testing.pytestArgs": [],
+    "python.testing.unittestEnabled": false,
+    "python.testing.pytestEnabled": true
+}
```

Comment:
We don't want the .vscode config to be committed to the repo. This stuff should probably be in the .gitignore. You can add it like [so](https://github.com/qiime2/qiime2/blob/95dad005ce10e8326f57877e120a17f47da27b47/.gitignore#L71)

Reply:
Thanks @Oddant1, I should have caught this! I will fix it ASAP

Reply:
It looks like this still needs to be removed from the diff since it was committed previously. The easiest thing to do is probably to just delete the `.vscode` folder locally and commit that removal. That shouldn't cause any problems on your end, vscode will just create the folder again when it needs to.

Reply:
I see, thanks for your help! I deleted the folder and committed the changes.